tests/mock_api/test_continuous_mastering.py

```diff
@@ -38,3 +40,12 @@ def test_continuous_mastering():
 
     op = project.published_clusters().refresh(poll_interval_seconds=0)
     assert op.succeeded()
+
+    estimate_url = (
+        f"http://localhost:9100/api/versioned/v1/projects/1/estimatedPairCounts"
+    )
+    estimate_json = {"isUpToDate": True}
+    responses.add(responses.GET, estimate_url, json=estimate_json)
+
+    status = project.estimate_pairs().is_up_to_date
```

Comment:
I'd prefer more unit-test centric validation, i.e. given a canned API response, ensure that each property / method on EstimatedPairCounts gives back the right results.